lib/node_modules/@stdlib/stats/base/dvarmpn/benchmark/benchmark.js

```diff
@@ -39,13 +45,7 @@ var dvarmpn = require( './../lib/dvarmpn.js' );
 * @returns {Function} benchmark function
 */
 function createBenchmark( len ) {
-	var x;
-	var i;
-
-	x = new Float64Array( len );
-	for ( i = 0; i < x.length; i++ ) {
-		x[ i ] = ( randu()*20.0 ) - 10.0;
-	}
+	var x = linspace( -len/2, len/2, len, options );
```

Comment:
```suggestion
	var x = uniform( len, -10.0, 10.0, options );
```
basically this.

you need to import `@stdlib/random/array/uniform`.

Reply:
Resolved